source/scss/components/_header-hotline.scss

```diff
@@ -0,0 +1,57 @@
+// HEADER-HOTLINE.SCSS
+
+.header-hotline {
+  align-items: center;
+  display: flex;
+  font-size: .93rem; // The icon-size is bound to the font-size
+  justify-content: flex-end;
+
+  // some CSS that gives the content the approximately the size it should have
+  > i::before {
+    display: block;
+    height: 17px;
+    transform: scale(1.2);
+    width: 16px;
+  }
+
+  // It's a p-tag, because the Typo3 Fluid contentElement creates a p-Tag.
+  > p {
+    display: flex;
+    flex-direction: column;
+    font-size: .75rem;
+    font-weight: 500;
+    margin-left: 5px;
+
+    .header-hotline-number {
+      font-weight: 600;
+    }
+  }
+
+  // p-tag related CSS, to correct the styles
+  > p {
```

Comment:
Yes, because I'd like to discuss what can be maintained by an editor and what not and with what kind of input fields. The p-tag is generated by the CMS which is none of our intention. Stil I have to consider it now, but I wish to get rid of it in the future. It is written like this to maintain it better.